lib/vagrant-vmck/version.rb

```diff
@@ -1,5 +1,5 @@
 module VagrantPlugins
   module Vmck
-    VERSION = "0.0.3"
+    VERSION = "0.0.4"
```

Comment:
No, let's make this change in a separate commit, after we merge into master.

Reply:
That's right, sorry, forgot it in here.